finish end to end db proxy endpoint class with tests

saraNoigershel opened the pull request.

riki7649255 reviewed (changes requested):
DB/NEW_KT_DB/DataAccess/DBProxyEndpointManager.py
1. _to_sql()- all the girls put it in the model, just letting you know
2. def select() => not an intuitive name, please rename, also: why def map_data_to_col_value_dict is inside o it? wrong indentation?

DB/NEW_KT_DB/Service/Classes/DBProxyEndpointService.py
3. remove commented code and all development leftovers if exist

saraNoigershel commented:
1. I deleted files and also added in gitignore that they won't upload anymore...
2.  The function select_objects_attributes_in_col_value_dict simply pulls from the table... it is a relatively generic function:
 you can send it name and then it sends values ​​of a certain object precisely according to the unique value of name. If not sent, pulls all the objects from the table
You can send it columns and then it pulls out certain columns in the table. If we don't send, everything is taken out.
gpt's have no good suggestion for a name for the function: 
select_db_proxy_endpoint_attributes , get_attributes and more, so In the meantime I left the previous name
but I added more documentation to the function to tell what it does.
3. I moved the inner function map_data_to_col_value_dict out
4. The function validate_tags get a list of tags and check if all tags in list are correct, so I think that it is the best name for it....
